packages/app/src/components/PageEditor/CommentMentionHelper.ts

```diff
@@ -32,14 +37,14 @@ export default class CommentMentionHelper {
     }
 
     // Get username after `@` character and search username
-    const mention = searchMention.substr(1);
+    const mention = searchMention.slice(1);
     this.editor.showHint({
       completeSingle: false,
       hint: async() => {
         if (mention.length > 0) {
           const users = await this.getUsersList(mention);
           return {
-            list: users.length > 0 ? users : [{ text: '', displayText: i18n.t('page_comment.no_user_found') }],
+            list: users.length > 0 ? users : [{ text: '', displayText: this.t('page_comment.no_user_found') }],
```

Comment:
Can we use `import { i18n } from 'next-i18next';`?

Reply:
Thank you, `i18n` is now worked